pkg/state/state.go

```diff
@@ -472,6 +470,9 @@ func (st *HelmState) ApplyOverrides(spec *ReleaseSpec) {
 		components := strings.Split(n, "/")
 
 		name = components[len(components)-1]
+		if !releaseInstalledInfo[name] {
+			st.logger.Warnf("WARNING: %s", fmt.Sprintf("release %s needs %s, but %s is not installed", spec.Name, name, name))
```

Comment:
Maybe guiding users to the next action would be nicer!

```suggestion
			st.logger.Warnf("WARNING: %s", fmt.Sprintf("release %s needs %s, but %s is not installed due to `installed: false`. Either mark %s as installed or remove %s from %s's `needs`", spec.Name, name, name, name, name, spec.Name))
```